en/mkdocs.yml

```diff
@@ -99,6 +99,8 @@ nav:
         - Service Orchestration: integration-guides/integration-as-api/service-orchestration.md
         - Message Transformation: integration-guides/integration-as-api/message-transformation.md
         - Message Routing: integration-guides/integration-as-api/message-routing.md
+      - Integration Service As a Managed API:
```

Comment:
Can we move this to under `usecases` ? 

<img width="1232" height="929" alt="image" src="https://github.com/user-attachments/assets/be9f2c3c-b4bd-4b8c-b489-7923d43332b1" />